.github/workflows/post-pr-cleanup.yml

```
@@ -0,0 +1,30 @@
+# This workflow takes care of cleaning up the pull request deploy for a clean S3 bucket.
+name: Post-PR cleanup
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+
+env:
+  BUILD_DEST_DIR: ${{ github.event.repository.name }}/${{ github.event.pull_request.number }}
+  S3_BUCKET: frontend-pulls
```

There is an existing secret for the bucket name.
```suggestion
  S3_BUCKET: ${{ secrets.AWS_S3_PR_DEPLOYMENT_BUCKET }}
```